src/main/webapp/resources/css/bootstrap-admin.min.css

```diff
@@ -1,7 +1,4971 @@
-/*!
```

Comment:
don't modify this file, if you need to add some new css, add them in `admin.css`

Reply:
ok, but @media is written in admin.min.css
if I want to write in admin.css it will be duplication styles

Reply:
@nayankumarrn21 you don't modify a min.css file.  You override the minimal amount of css in the `admin.css` file.

Reply:
you may have not noticed, but the `bootstrap-admin.min.css` has been re-formatted totally, I can't do a code review if all the file has changed.